spec/features/application_record_creation_spec.rb

```diff
@@ -14,6 +14,7 @@
     before :each do
       when_current_user_is student, integration: true
       visit application_path(application_template)
+      application_template.position.update(not_hiring_text: 'custom text')
```

Comment:
I think you may have interpreted my recommendation lol
You may already know this, but with rspec you can register a before :each hook for each nested context block you make. This allows you to sort of keep getting more specific with nested contexts.

To do what I recommended earlier you would create a new before :each hook, within the `'application template has been marked as inactive' context` block.

Then you would take these three lines and throw them in that new before :each hook

`application_template.update active: false`
`application_template.position.update(not_hiring_text: 'custom text')`
`visit current_url`